ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/DdlCommandExec.java

```diff
@@ -37,26 +37,28 @@ public DdlCommandExec(MetaStore metaStore) {
   /**
    * execute on temp metaStore
    */
-  public DdlCommandResult tryExecute(DdlCommand ddlCommand, MetaStore tempMetaStore) {
+  public DdlCommandResult tryExecute(DdlCommand ddlCommand, MetaStore tempMetaStore,
```

Comment:
Should isValidatePhase ever be false here? We would only run against a temp meta store if we are validating right? If that's the case we should get rid of that parameter. That would address my concern with buildSingleQueryAst

Reply:
Yes, it will always bee false. I removed this.